src/main.rs

```diff
@@ -263,3 +268,26 @@ fn shutdown() -> eyre::Result<impl std::future::Future<Output = ()>> {
         }
     }))
 }
+
+fn init_tracing() -> eyre::Result<()> {
+    let fmt = tracing_subscriber::fmt::layer().with_ansi(stdout().is_terminal());
+
+    let env = match std::env::var("RUST_LOG") {
+        Ok(env) => env,
+        Err(std::env::VarError::NotPresent) => {
+            "e2e_test=debug,astarte_message_hub=debug".to_string()
```

Comment:
I don't know if it is right here to refer also `e2etest`, the src/main shouldn't know the `e2etst`.